akka-actor/src/main/scala/akka/util/Throwables.scala

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+package akka.util
+
+import scala.util.control.NonFatal
+
+object Throwables {
```

Comment:
needs docs why this exists and what the intended use case is -- to allow catching only non fatal ones in supervision on routing in akka http I suppose